CONTRIBUTING.md

```diff
@@ -28,15 +28,46 @@ In order to improve the trust and security of the code contributed we require th
 
 Our editor of choice is Visual Studio Code (VS Code), it is not required for working with the code, but is recommended for sharing code and contributing with others.
 
-## Installing dependencies
-
-Once you have cloned the repository, to install the dependencies run:
-
-```shell
-npm install --save-dev
-```
-
-This will install all of the needed dependencies.
+## Setting up the project
+
+1. Clone the repository
+
+    ```shell
+    git clone https://github.com/torontojs/vms.git
+    cd vms
+    ```
+
+2. Install Volta to manage your node version and pin a specific version of `node` to the project.
+    - Note: If you already have `nvm` installed, `volta` will conflict with it so it's best to uninstall `nvm`.
+    ```shell
+        curl https://get.volta.sh | bash
+        source ~/.bashrc
+    ```
+
+3. Install the dependencies
+    ```shell
+        npm install
+    ```
+
+4. Copy the `.dev.vars.example` file to `.dev.vars` for providing environment secrets during development.
+    ```shell
+        cp .dev.vars.example .dev.vars
+    ```
+5. Create a SendGrid account and get your API key. We will use this to send emails.
```

Comment:
Creating a sendgrid account should be on the previous section as one of the prerequisites

Reply:
Ok that makes sense -- will update later